tests/integration/components/frost-tab-test.js

```diff
@@ -51,6 +51,7 @@ describe(test.label, function () {
   })
 
   it('Renders', function (done) {
+    this.timeout(5000)
```

Comment:
This should be written as a `describe` with a `beforeEach()` where the `setProperties()` and `render()` are handled and the `expects()` are inside the `it()`. If needed you can then put the `return wait()` in the `beforeEach`.

Reply:
```
describe('when something is done', function() {
  beforeEach(function () {
    this.setProperties({})
    this.render()
    return wait()
  })

  it('should have expected outcome', function () {
   expect()
  })
})
```